### ISSUE TITLE
[BUG] MS SQL Token error: 'Invalid object name 'information_schema.schemata'.'

### ISSUE THREAD

**Issue by kodaicoder** [DataflareApp/Dataflare#35]
### Describe the bug

On testing connection is successful
![image](https://github.com/DataflareApp/Dataflare/assets/110652592/b2aa316b-2473-4b7b-9aee-0feab20bcba2)

![image](https://github.com/DataflareApp/Dataflare/assets/110652592/ff640465-db3d-44f0-be99-4d686b9c66e0)

but after press connect button I got error
Token error: 'Invalid object name 'information_schema.schemata'.' on server SERVER_NAME executing on line 2 (code: 208, state: 1, class: 16);

![image](https://github.com/DataflareApp/Dataflare/assets/110652592/05d52b68-82ba-41e2-a810-5107bc93072c)

btw if I connecting with SSMS to remote server is still working

### Platform and Database

```text
Windows 11 
version 23H2
OS Build 22635.2915

Microsoft SQL Server Standard Edition (64-bit)
Version 13.0.1601 RTM
```

**Comment by wyhaya** [comment]
@kodaicoder  Thanks.

Dataflare gets the schema and table through this SQL, what is certain is that the problem comes from this piece of SQL, maybe a compatibility issue, I'll look into it.

```sql
SELECT
    c.schema_name,
    t.table_name
FROM
    information_schema.schemata as c
    LEFT JOIN information_schema.tables as t ON
        t.TABLE_TYPE = 'BASE TABLE' AND
        t.table_schema = c.schema_name
ORDER BY c.schema_name, t.table_name;
```

**Comment by kodaicoder** [comment]
I has trying change the code you mention into all upper case like this
```
SELECT
    c.SCHEMA_NAME,
    t.TABLE_NAME
FROM
    INFORMATION_SCHEMA.SCHEMATA as c
    LEFT JOIN INFORMATION_SCHEMA.TABLES as t ON
        t.TABLE_TYPE = 'BASE TABLE' AND
        t.TABLE_SCHEMA = c.SCHEMA_NAME
ORDER BY c.SCHEMA_NAME, t.TABLE_NAME;
```

and it working good 
BTW I don't know if it will affected and cause error on any other MSSQL version or other SQL tools

**Comment by wyhaya** [comment]
@kodaicoder @wqweto 
We have just released version `0.6`, and this error has been fixed. Can you please try again?

**Comment by wqweto** [comment]
It does work ok now!

Btw, on browsing tables the arrow in the cells for following foreign key is a nice touch :-))
